### PR TITLE
LoadInfrastructure

### PR DESCRIPTION
**Pull request by LePetitTim**
Rename LoadSignage by LoadInfrastructure.
Add possibility to put Signage or Infrastructure with type : Building, facility, signage
Create tests for loadsignage (loadinfrastructure) for signage and Infrastructure with datas

Check this pull request #1630 
Was done only for signage and was not test

### CODE REVIEW

**Comment by gutard** [comment]
Il serait plus explicite de rajouter une option --signage plutôt que --type=signage et remplacer --label par --type.

File: geotrek/infrastructure/management/commands/loadinfrastructure.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+
+class Command(BaseCommand):
+    help = 'Load a layer with point geometries in te structure model\n'
+    can_import_settings = True
+    counter = 0
+
+    def add_arguments(self, parser):
+        parser.add_argument('point_layer')
+        parser.add_argument('--signage', dest='signage', action='store_true', help='Base url')
```
Comment (gutard) [inline geotrek/infrastructure/management/commands/loadinfrastructure.py]: Pourquoi help='Base url' ?

File: geotrek/infrastructure/management/commands/loadinfrastructure.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+        if created:
+            self.stdout.write(u"- Condition Type '{}' created".format(condition_type))
+
+        structure, created = Structure.objects.get_or_create(name=structure)
+
+        if created:
+            self.stdout.write(u"- Structure '{}' created".format(structure))
+        with transaction.atomic():
+            if model == 'S':
```
Comment (gutard) [inline geotrek/infrastructure/management/commands/loadinfrastructure.py]: Pour moins dupliquer de code on peut écrire :
```
Model = Signage if model == 'S' else Infrastructure
Model.objects.create(…)
```

**Comment by camillemonchicourt** [comment]
This is a great news, but not sure how to use it. 
Would be great to have a few lines about the import commands (POI, infrastructures, from SHP only ?) into https://github.com/GeotrekCE/Geotrek-admin/blob/master/docs/import.rst

**Pull request merged**